<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat_Window</class>
 <widget class="QMainWindow" name="Chat_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>353</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>215</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Чат</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_chat">
              <property name="maximumSize">
               <size>
                <width>800</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Чат:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_status">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="Chat">
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>800</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_users">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>сейчас в сети:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="user_list">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="text_to_mes">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>811</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="send_btn">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>отправить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>меню</string>
    </property>
    <addaction name="new_accaunt_action"/>
    <addaction name="change_account_action"/>
    <addaction name="action_ab_window"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>авторизация</string>
   </property>
  </action>
  <action name="action_ab_window">
   <property name="text">
    <string>о программе</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="new_accaunt_action">
   <property name="text">
    <string>новый аккаунт</string>
   </property>
  </action>
  <action name="change_account_action">
   <property name="text">
    <string>сменить аккаунт</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_3</sender>
   <signal>triggered()</signal>
   <receiver>Chat_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
